test/Microsoft.AspNetCore.Razor.Evolution.Test/DefaultDirectiveIRPassTest.cs

```diff
@@ -1,171 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
Ya this guy shouldn't have been terminated.
![image](https://cloud.githubusercontent.com/assets/2008729/23927012/b3161bfa-08d4-11e7-9d7b-6726530ea8c8.png)